pandas/tests/extension/base/base.py

```diff
@@ -1,9 +1,11 @@
+from typing import Any
+
 import pandas.util.testing as tm
 
 
 class BaseExtensionTests:
 
     assert_equal = staticmethod(tm.assert_equal)
-    assert_series_equal = staticmethod(tm.assert_series_equal)
-    assert_frame_equal = staticmethod(tm.assert_frame_equal)
+    assert_series_equal = staticmethod(tm.assert_series_equal)  # type: Any
```

Comment:
why is this needed?

Reply:
I'm not clear on why these need to be staticmethods at all